src/search/components/SearchResultItem.tsx

```diff
@@ -100,15 +125,15 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 						result.administrative_type
 					)}
 				</SearchResultTitle>
-				{!!result.original_cp && (
-					<SearchResultSubtitle>
-						{renderSearchLink(
-							result.original_cp,
-							{ filters: { provider: [result.original_cp] } },
-							'c-body-2'
-						)}
-					</SearchResultSubtitle>
-				)}
+				<SearchResultSubtitle>
+					{result.original_cp
+						? renderSearchLink(
+								result.original_cp,
+								{ filters: { provider: [result.original_cp] } },
+								'c-body-2'
+						  )
+						: renderAuthorOrOrganisation(result)}
```

Comment:
this seems to be a bit double with the result.original_cp check in renderAuthorOrOrganisation
maybe we need to split that part off into a function: renderCpLink?